apps/admin/application.rb

```diff
@@ -258,9 +258,16 @@ class Application < Hanami::Application
       # This is useful for sharing common functionality
       #
       # See: http://www.rubydoc.info/gems/hanami-controller#Configuration
+      ADMIN_USERNAME = ENV.fetch('ADMIN_USERNAME')
+      ADMIN_PASSWORD = ENV.fetch('ADMIN_PASSWORD')
```

Comment:
good idea, WDYT if we replace it to initialize file? (I can do it after merge :) )

Reply:
Yes that's a very good idea 👍.